astyled_whitelist

```diff
@@ -1,5 +1,4 @@
 src/action.cpp
-src/activity_handlers.h
```

Comment:
Good question, it bugged me too and I should have addressed this in the description.
Short answer, because it was a redundant entry, there's another instance of it further down in the file.
Longer answer, astyled_whitelist is ordered first by .cpp files vs .h files, then ordered alphabetically.  If you don't update using the procedure outlined in docs/CODE_STYLE.txt, you're likely going to add entries in the wrong order, but that doesn't mater much other than this random-looking shuffling of lines the next time someone does use that procedure.